software/spmd/common/link_dram.ld

```diff
@@ -196,6 +196,8 @@ SECTIONS
 
     _dram_misc = .;
     *(.*)
+
+    . = ALIGN(4);
```

Comment:
why is this align necessary? (comments are a good thing)

Reply:
Oh this, I don't think this line is required. But, without this line loading binaries linked with this script to cosim/f1 was not possible. Thought of discussing with you guys but forgot to, thanks for pointing it out!